report/main.py

```diff
@@ -0,0 +1,476 @@
+"""Python Script: Build a report for email attachment"""
+import base64
+from datetime import datetime, timedelta
+from os import environ, _Environ
+
+import altair as alt
+from altair.vegalite.v5.api import Chart
+from dotenv import load_dotenv
+import pandas as pd
+from pandas import DataFrame
+from psycopg2 import connect
+from psycopg2.extensions import connection
+
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.application import MIMEApplication
+
+from xhtml2pdf import pisa
+import plotly_express as px
+import boto3
```

Comment:
from my understanding, these should not be separated with newline

Reply:
Oh yes, in my html induced depression I forgot to format this! Will normalise.